packages/base/src/commands.ts

```diff
@@ -49,6 +51,23 @@ export function newName(type: string, model: IJupyterCadModel): string {
   return name;
 }
 
+export async function dryRunCheck(options: {
```

Comment:
How about we create an interface for options: 
```javascript
interface DryRunOptions {
  jcadContent: IJCadContent;
  mainView: MainViewModel;
  requestedOperator: string;
}
```
& use it as `dryRunCheck(options: DryRunOptions)`?

Reply:
I would start creating an interface once it's used twice.

Reply:
Right, that's fair.